OpenSim/Examples/ExampleHopperDevice/buildHopperModel.cpp

```diff
@@ -88,15 +88,16 @@ Model buildHopper() {
 
     // Set the coordinate names and default values. Note that we need "auto&"
     // here so that we get a reference to the CoordinateSet rather than a copy.
-    auto& sliderCoord = sliderToGround->upd_CoordinateSet()[0];
+    auto& sliderCoord =
```

Comment:
Maybe the PR you fixed it in has not been merged yet. The master currently has this problem:
https://github.com/opensim-org/opensim-core/blob/master/OpenSim/Sandbox/ExampleHopperDevice/buildHopperModel.cpp#L91